src/constants/endpoints.ts

```diff
@@ -0,0 +1,18 @@
+export default {
+  LOGIN: '/api/v1/token/',
+  SIGN_UP: '/api/v1/users/',
+  EDIT_USER: '/api/v1/users/:id/',
+  ME: '/api/v1/users/me/',
+  LOGOUT: '/auth/logout',
+  FACEBOOK: '/api/v1/social/facebook',
+  GOOGLE: '/api/v1/social/google',
+  REFRESH_TOKEN: '/api/v1/token/refresh/',
+  START_PASSWORD_RECOVERY: '/api/v1/password_reset/',
+  VALIDATE_PASSWORD_RECOVERY_TOKEN: '/api/v1/password_reset/validate_token/',
+  CONFIRM_PASSWORD_RECOVERY: '/api/v1/password_reset/confirm/',
+  FORGOT_PASSWORD: '/user/forgot-password',
+  RESET_PASSWORD: '/api/v1/password_reset/',
```

Comment:
This one duplicates `START_PASSWORD_RECOVERY: '/api/v1/password_reset/',` so should be removed
```suggestion
  RESET_PASSWORD: '/api/v1/password_reset/',
```